middleware/auth.js

```diff
@@ -0,0 +1,27 @@
+require("dotenv");
+const jwt = require("jsonwebtoken");
+const User = require("../models/user.model");
+const secret = process.env.SECRET;
+
+const createToken = async (data) => {
```

Comment:
`data` is too generic, please find a variable name that telling the story better